app/src/main/java/ch/epfl/sdp/musiconnect/Musician.java

```diff
@@ -1,16 +1,19 @@
 package ch.epfl.sdp.musiconnect;
 
+import androidx.room.Entity;
+
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Set;
 
 /**
  * @author Manuel Pellegrini, EPFL
  */
+@Entity
 public class Musician extends Person implements Performer {
 
     private String videoURL;
-    private Map<Instrument, Level> instruments;
+    public Map<Instrument, Level> instruments;
 
```

Comment:
Is there not a getter instead?

Reply:
room had problem with these fields; making them public ensured it worked. I'll try to see if I can make a getter work